src/ripple/app/ledger/Ledger.cpp

```diff
@@ -209,9 +209,9 @@ Ledger::Ledger (uint256 const& parentHash,
                 beast::Journal j)
     : mImmutable (true)
     , txMap_ (std::make_shared <SHAMap> (
-        SHAMapType::TRANSACTION, transHash, family))
-    , stateMap_ (std::make_shared <SHAMap> (
-        SHAMapType::STATE, accountHash, family))
+        SHAMapType::TRANSACTION, transHash, family, SHAMap::version{1}))
+    , stateMap_ (std::make_shared <SHAMap> (SHAMapType::STATE, accountHash,
```

Comment:
I'm mad as Hell and I'm going to this this anymore!
https://www.youtube.com/watch?v=rGIY5Vyj4YM
:-)